idaplugin/rematch/actions/match.py

```diff
@@ -171,24 +182,86 @@ def perform_task(self):
       progress = int(r['progress'])
       status = r['status']
       if status == 'failed':
-        self.pbar.reject()
+        self.pbar.cancel()
```

Comment:
Yes. This is quite big and not really logged. I'll add logging throughout the code. Good idea!